pac4j-saml/src/main/java/org/pac4j/saml/metadata/Saml2MetadataFilter.java

```diff
@@ -0,0 +1,79 @@
+package org.pac4j.saml.metadata;
+
+import java.io.IOException;
+
+import javax.servlet.Filter;
+import javax.servlet.FilterChain;
+import javax.servlet.FilterConfig;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+
+import org.pac4j.core.config.Config;
+import org.pac4j.core.exception.TechnicalException;
+import org.pac4j.core.util.CommonHelper;
+import org.pac4j.saml.client.SAML2Client;
+
+/**
+ * This filter prints the SP metadata for SAML.
+ * 
+ * Example shiro.ini configuration:
+ * 
+ * saml2MetadataFilter = org.pac4j.saml.metadata.Saml2MetadataFilter
+ * saml2MetadataFilter.config = $config
+ * saml2MetadataFilter.client = SAML2Client
+ * 
+ * [urls]
+ * /API/SAML2/metadata = saml2MetadataFilter
+ * 
+ * @author Graham Leggett
+ * @since 3.8.0
+ */
+public class Saml2MetadataFilter implements Filter {
+
+    private Config config;
+
+    private String client;
```

Comment:
Change the property name as well: `clientName`...

Reply:
It's been done.